crates/core/src/test.rs

```diff
@@ -6330,6 +6330,41 @@ fn inline_list_length() {
     .unwrap();
 }
 
+#[test]
+fn test_delete_hanging_comma() {
+    run_test_expected(TestArgExpected {
+        pattern: r#"
+            |language python
+            |
+            |`TaskMetadata($args)` => `TaskMetadata($args)`
+            |"#
+        .trim_margin()
+        .unwrap(),
+        source: r#"
+            |TaskMetadata(
+            |    name="TestTask",
+            |    description="This is a test task.",
+            |    type="TestType",
+            |    ,
```

Comment:
This isn't a representative test. We specifically want to test cases where an argument is *removed*, and therefore the trailing comma after it is also removed.

Reply:
@morgante I was also working on the issue and recently pushed my [PR ](https://github.com/getgrit/gritql/pull/417) .